src/components/button/button.scss

```diff
@@ -14,43 +14,43 @@
     cursor: default;
   }
 
-  &.dark {
-    @include blue-hover;
-    border-color: $light-blue;
-    background-color: $blue;
-    color: $white;
-
-    &:disabled {
-      color: $light-blue;
-      background-color: $dark-grey-blue;
-    }
+  &.medium {
+    padding: 20px 30px;
+    border-radius: 10px;
   }
+}
 
-  &.light {
-    @include white-hover;
-    background-color: $white;
-    border-color: $blue;
-    color: $blue;
-
-    &:disabled {
-      border-color: $grey;
-      color: $grey;
+.dark-button {
```

Comment:
changes here cause the selector specificity to change breaking a lot of hover effects across the app such as
  - account popup buttons
  - bubble dropdown
  - ...and probably more

i'm assuming there was a reason this was done so that custom styles worked on the search page tiles.
a solution could be to fix all those areas affected but i'm not sure about the breadth of it beyond those

perhaps it would be better to move these selectors back to the way they were (nested inside `.button`) and then you could be more specific in your custom styles (e.g. `.button.deck-tile` instead of `.deck-tile`) to get the styles you want